2025-10-31 03:23:04,654 - WARNING - Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002513025A7B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /api/2.0/mlflow/registered-models/get-latest-versions
2025-10-31 03:23:13,568 - WARNING - Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000251302465D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /api/2.0/mlflow/registered-models/get-latest-versions
2025-10-31 03:23:26,010 - WARNING - Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025130246850>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /api/2.0/mlflow/registered-models/get-latest-versions
2025-10-31 03:23:46,465 - WARNING - Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025130125F30>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /api/2.0/mlflow/registered-models/get-latest-versions
2025-10-31 03:24:22,551 - WARNING - Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025130126060>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /api/2.0/mlflow/registered-models/get-latest-versions
2025-10-31 03:25:30,684 - WARNING - Retrying (Retry(total=1, connect=1, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002513014B2F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /api/2.0/mlflow/registered-models/get-latest-versions
2025-10-31 03:27:20,670 - INFO - Loaded production model: IrisClassifier_Sklearn
2025-10-31 03:27:30,697 - INFO - Loaded production model: IrisClassifier_PyTorch
2025-10-31 03:27:30,697 - INFO - Models loaded successfully during startup.
2025-10-31 03:28:46,513 - INFO - Application shutting down... cleanup if needed
